### ISSUE TITLE
Custom pre-trained models

### ISSUE THREAD

**Issue by applepingping** [MC-E/DragonDiffusion#28]
Thank you for your excellent work! I want to know how to load my own pre-trained model?

**Comment by MC-E** [comment]
Could you provide more details of your own model?

**Comment by applepingping** [comment]
Thank you for your reply! Since my dataset is quite unique and there are no publicly available pre-trained models, I would like to conduct some fine-tuning methods such as full finetuning 、LORA fine-tuning and so on based on stable-diffusion-v1-5 to generate a safetensors model. I am not sure if your model supports this addition. If it does, could you please let me know how to proceed? Looking forward to your response!

**Comment by MC-E** [comment]
Yes, this framework supports custom models. Our pipeline is based on diffusers. You can replace the base model or insert LoRA according to your demand.

**Comment by applepingping** [comment]
Thank you very much for your reply! Do you support the input of the unconditional diffusion model? I don't really need the Vincennes model.

**Comment by applepingping** [comment]
Thank you very much for your answer! I have trained a fully tuned safetensors file and a LoRA tuned safetensors file, where should I add them? I would appreciate it if you could help me with that. Looking forward to your reply!
<img width="1224" alt="image" src="https://github.com/user-attachments/assets/2e20d946-0077-48ad-a4ee-cecaf96aca07">
<img width="1232" alt="image" src="https://github.com/user-attachments/assets/2376e41c-fa0d-46a8-980b-525681eadc41">
